code/modules/projectiles/projectile/bullets.dm

```diff
@@ -124,6 +124,16 @@
 	damage = 10
 	weaken = 4
 	stun = 4
+	
+/obj/item/projectile/bullet/honker
+	damage = 0
+	weaken = 5
+	stun = 5
+	forcedodge = 1
+	hitsound = 'sound/items/bikehorn.ogg'

```

Comment:
I fall into the conservative wing of the Clown coalition.

If I put it into guns, the firesound is always an airhorn.